### PR TITLE
test(InfoTip): Update InfoTip tests for more complete coverage

### PR DESCRIPTION
**Pull request by dmiller9911**
Updated InfoTip tests to cover component functionality better.  Switched to use shallow render, and
avoided calling instance methods directly. Added test suites for each individual component instead
of 1 test suite for all.

re #287

**What**:

Updated InfoTip tests to better align with testing best practices with Enzyme.  This PR can serve as an additional discussion point for #287 as we all learn the best way to test the React components within this repo.

**Link to Storybook**:
N/A

**Additional issues**:
#287

### CODE REVIEW

File: src/components/InfoTip/InfoTip.js [hunk 1]
```diff
@@ -44,24 +44,19 @@ class InfoTip extends React.Component {
...
+  };
 
-  handleBackFocus() {
-    if (this.state.open) {
-      this.setState({ open: false });
-    }
-  }
-
-  handleClick(event) {
+  handleClick = event => {
```
Comment (jeff-phillips-18) [inline src/components/InfoTip/InfoTip.js]: I though we standardized on using the bindMethods call? I don't mind this (actually prefer it) but I thought the decision was made to use the helper.

Comment (dmiller9911) [reply src/components/InfoTip/InfoTip.js]: Ah was unaware of a standardization.  I am cool to switch back if you want. Either way is essentially the same thing. The only benefit to this is typing and it matches [spec (stage-3)](https://github.com/tc39/proposal-class-fields).  Let me know thoughts on if I should switch it back.

Comment (jeff-phillips-18) [reply src/components/InfoTip/InfoTip.js]: We should probably remain consistent here and update all instances if/when we decide to go that route (I would support that)

Comment (dmiller9911) [reply src/components/InfoTip/InfoTip.js]: Sounds good.  Opened up an issue to track that change: #294

File: src/components/InfoTip/InfoTip.test.js [hunk 2]
```diff
@@ -1,229 +1,119 @@
...
+import { Dropdown } from '../Dropdown';
+
+const props = {
+  id: 'InfoTip',
+  children: 'children'
+};
+
+test('renders Dropdown', () => {
+  const view = shallow(<InfoTip {...props} />);
+  expect(view).toMatchSnapshot();
```
Comment (priley86) [inline src/components/InfoTip/InfoTip.test.js]: same question - do you want to name these or just leave the name argument out ;) ?

**Pull request merged**